src/tranche/tranche-space.rain

```diff
@@ -317,11 +317,11 @@ charts:
         - label: Starting tranche
           value: 0.2.0
           description: 'This strategy starts executing sells at the initial tranche (stack item 0.2.0)'          
-        - label: Initial sell price
-          value: 0.5.3
+        - label: Initial effective sell price
```

Comment:
this is the final literal io ratio, it's not a derived price

Reply:
yes correct, for sell order final io ratio is the effective sell price. Do you mean to say I remove the word 'effective' ?

Reply:
resolved.